core/modules/replica/VerifyApp.cc

```diff
@@ -0,0 +1,139 @@
+/*
+ * LSST Data Management System
+ * Copyright 2018 LSST Corporation.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+// Class header
+#include "replica/VerifyApp.h"
+
+// System headers
+#include <atomic>
+#include <iomanip>
+#include <iostream>
+#include <vector>
+
+// Qserv headers
+#include "replica/Configuration.h"
+#include "replica/Controller.h"
+#include "replica/ReplicaInfo.h"
+#include "replica/VerifyJob.h"
+#include "util/BlockPost.h"
+
+using namespace std;
+
+namespace {
+
+string const description {
+    "This application runs the replica verification algorithm for all known"
+    " replicas across all ENABLED workers"
```

Comment:
Add dot at the end of sentence (in many other places too).

Reply:
Ok